packages/shared-services/src/notification-service.ts

```
@@ -0,0 +1,86 @@
+
+  markAsRead(id: string): boolean {
+    const notification = this.notifications.find(n => n.id === id);
+    if (notification) {
+      notification.read = true;
+      return true;
+    }
+    return false;
+  }
+
+  markAllAsRead(): void {
+    this.notifications.forEach(n => n.read = true);
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)